tutorials/migration_sdf.md

```diff
@@ -282,14 +282,12 @@ within a `<script>` aren't supported on Gazebo, for example:
         </material>
 ```
 
+Although, there is limited parsing capabilities available in Gazebo to support pre-defined materials.
 To make your models compatible with both simulators, you can use these alternatives:
 
 ### Plain colors
 
-If the material defines plain colors, use the `<ambient>`, `<specular>`,
-`<emissive>` and `<diffuse>` tags as needed.
-
-For example, this material from
+Plain solid colors are automatically parsed if encountered, for example, this material from
```

Comment:
This makes it sound like solid colors from other material files are supported, which is not the case. Can you clarify that it is only the `gazebo.material` file shipped with Gazebo that is supported?